packages/dapp/env.d.ts

```diff
@@ -1,12 +1,19 @@
 /// <reference types="vite/client" />
 
 interface ImportMetaEnv {
+  readonly BASE_URL: string;
+  readonly DEV: boolean;
+  readonly MODE: 'development' | 'testnet' | 'production' | 'staging';
+  readonly PROD: boolean;
+  readonly SSR: boolean;
```

Comment:
Is this unnecessary? Still running into typescript issues on those env´s! 

1.) I guess overwriting such as BASE_URL, DEV, PROD or SSR does not make much sense, right? It is taken from the Vite´s ImportMetaEnv.
2.) But does it make sense to define the MODE types? Or is it bad practise?

Here is the error I wish to get fixed:
![image](https://user-images.githubusercontent.com/43956540/232039859-77a69ca9-17c0-4387-870a-585793e2c19f.png)

Reply:
ℹ️ Of course, same applies also the the develop-branch. This PR here is only being merged into the testnet-branch. So, any improvements need to be done to develop as well.